schema-source/RSxxSub.schema.yaml

```diff
@@ -0,0 +1,224 @@
+Schema:
+  Object Type: "Meta"
+  References:
+    - "ASHRAE205"
+  Title: "Fenestration - Glazing Substrate"
+  Description: "Schema for ASHRAE 205 annex RSxxSub: Fenestration - Glazing Substrate" 
+  Version: "2.0.0"
+  Root Data Group: "RSxxSub"
+
+# Data Groups
+RSxxSub:
+  Object Type: "Data Group"
+  Data Elements:
+    metadata:
+      Description: "Metadata data group"
+      Data Type: "{Metadata}"
+      Constraints: "schema=RSxxSub"
+      Required: True
+    description:
+      Description: "Data group describing product and rating information"
+      Data Type: "{Description}"
+    performance:
+      Description: "Data group containing performance information"
+      Data Type: "{Performance}"
+      Required: True
+
+Description:
+  Object Type: "Data Group"
+  Data Elements:
+    product_information:
+      Description: "Data group describing product information"
+      Data Type: "{ProductInformation}"
+
+ProductInformation:
+  Object Type: "Data Group"
+  Data Elements:
+    manufacturer:
+      Description: "Manufacturer name"
+      Data Type: "String"
+    model_number:
+      Description: "Model number"
+      Data Type: "Pattern"
+      Notes: "Pattern shall match all model numbers that can be represented by the representation"
+
+Performance:
+  Object Type: "Data Group"
+  Data Elements:
+    thickness:
+      Description: "The total thickness of the glazing."
+      Data Type: "Numeric"
+      Units: "m"
+      Required: True
+    thermal_conductivity:
+      Description: "Overall thermal conductivity of the glazing."
+      Data Type: "Numeric"
+      Units: "W/m-K"
+      Required: True
+    front_side_infrared_emissivity:
+      Description: "Front Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: True
+      Constraints: [">=0.0", "<=1.0"]
+    back_side_infrared_emissivity:
+      Description: "Back Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: False
+      Constraints: [">=0.0", "<=1.0"]
+      Notes: "When this data element is missing, the value is assumed to be the same as the front_side_infrared_emissivity."
+    is_coated:
```

Comment:
Is that what you want? Or do you want an option to define that this coated pane consists of a substrate with UUID1 and the coating UUID2?

Reply:
I'm trying to allow both the ability to define multiple levels where the film or coating is one level and the glazing substrate is another level or if the properties are only known together allow for the definition of a glazing substrate that includes the film or coating. That is what the Performance data group allows a list of Level data group children.